platform/core.multitabs/src/org/netbeans/core/multitabs/ButtonFactory.java

```diff
@@ -47,7 +48,11 @@
      */
     public static JButton createDropDownButton( Controller controller ) {
         final JButton btn = new JButton( new TabListPopupAction(controller) );
-        btn.setIcon( ImageUtilities.loadImageIcon( "org/netbeans/core/multitabs/resources/down.png", true) ); //NOI18N
+        Icon icon = UIManager.getIcon("nb.multitabs.button.dropdown.icon");
+        if (icon == null) {
+            icon = ImageUtilities.loadImageIcon( "org/netbeans/core/multitabs/resources/down.png", true ); //NOI18N
```

Comment:
This is just a fallback, right? I assume all of these icons are vector-drawn to work with HiDPI screens?

Reply:
No, not just a fallback.
Vector icons are only used for FlatLaf.

Other LaFs still use the bitmap images.

Reply:
But should be not very difficult to extend `Windows8VectorTabControlIcon` and `AquaVectorTabControlIcon` in the same way I've extended `FlatTabControlIcon`...